WBooks/Model/User.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  User.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 26/08/21.
+//
+
+import Foundation
+
+internal struct User: Codable {
```

Comment:
The default access level is internal. Are you adding for some reason? If it is not the case, you can remove it.

Reply:
Done